lib/src/network/coap_network_openssl.dart

```diff
@@ -6,144 +8,129 @@
  * Copyright :  Jan Romann
  */
 
+import 'dart:async';
 import 'dart:io';
 import 'dart:typed_data';
 
 import 'package:dtls/dtls.dart';
 import 'package:typed_data/typed_data.dart';
 
 import '../event/coap_event_bus.dart';
-import '../net/coap_internet_address.dart';
 import 'coap_inetwork.dart';
+import 'coap_network_udp.dart';
 
 /// DTLS network using OpenSSL
-class CoapNetworkOpenSSL implements CoapINetwork {
+class CoapNetworkUDPOpenSSL extends CoapNetworkUDP {
```

Comment:
A bit verbose naming, but when we have one DTLS solution for all cases in the future we could call it `CoapNetworkDTLS` instead, which implies UDP clearly.

Reply:
Since the Network classes are only used internally, maybe we could also use names like `DtlsNetworkOpenSSL` (or `NetworkDtlsOpenSSL`)?

Reply:
Agreed, I made a separate issue so we could go over the internal classes and do it to all of them to keep things streamlined: https://github.com/shamblett/coap/issues/121